FaaS/OpenFaaS/template/README.md

```diff
@@ -0,0 +1,52 @@
+# OpenFaaS template based on Clear Linux
+
+## How to use the template
+
+Assume you already had deployed workable OpenFaaS on top of Kubernetes.
+
+Steps could refer to
+
+[deployment-guide-for-kubernetes](https://docs.openfaas.com/deployment/kubernetes/#deployment-guide-for-kubernetes)
+
+[OpenFaaS workshop](https://github.com/openfaas/workshop)
+
+### python3-clearlinux
+1.  `mkdir test && cd test`
+2.  `cp -r "<your-dockerfiles-path>"/FaaS/OpenFaaS/template/. template/`
+3.  `faas-cli new --lang python3-clearlinux hello-openfaas --prefix="<your-docker-username-here>"`
+
+    Files tree as below.
+>  
+    ├── hello-openfaas
+    │   ├── bundles.txt
+    │   ├── handler.py
+    │   ├── __init__.py
+    │   └── requirements.txt
+    ├── hello-openfaas.yml
+    └── template
+
+
+    *  Put the required python packages in the "requirements.txt".
+    For example,
+
+    `echo "redis" >> hello-openfaas/requirements.txt`
```

Comment:
It seems these optional files (empty file) also submit together. Is there more better way handle this case? @bryteise

Reply:
Sorry I had missed this. I don't think I quite understand your question though.

Reply:
Oh, I meaning 5 empty configuration file is submitting with other changes together. Such as, requirements.txt; bundles.txt .... Not sure if this is better solution? or if you have better idea?

Reply:
@ZhongbaoShi I think the empty files are fine. Users who use  the template can add what they want into the requirements.txt or bundles.txt.

Reply:
I agree with @qzheng527, it isn't really a problem I don't think.